Sources/UIKitBackend/KeyboardToolbar.swift

```diff
@@ -0,0 +1,180 @@
+import SwiftCrossUI
+import UIKit
+
+/// An item which can be displayed in a keyboard toolbar. Implementers of this do not have
+/// to implement ``SwiftCrossUI/View``.
```

Comment:
This doc comment should make it clear that toolbar items are stateless and `@State` properties within conforming types will not act as expected when used in a keyboard toolbar body.

Reply:
Added a paragraph explaining this